src/Components/test/E2ETest/Tests/EventTest.cs

```diff
@@ -167,7 +167,7 @@ public void PreventDefault_DotNotApplyByDefault()
         }
 
         [Fact]
-        [Flaky("https://github.com/dotnet/aspnetcore-internal/issues/1987", FlakyOn.AzP.Windows)]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/1987")]
```

Comment:
Should we removing this parameters (or making it optional?). I've seen your boy @anurse passing in a disclaimer that we're not using issues for this anymore.

Reply:
NVM I see you dids't that. Top cheddar 🧀